Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -229,13 +231,13 @@ extension PreplannedMapModel: Hashable {
         lhs === rhs
     }
     
-    public func hash(into hasher: inout Hasher) {
+    nonisolated public func hash(into hasher: inout Hasher) {
```

Comment:
This class is internal, so neither of these methods (`==` and `hash(into:)`) should be public. Also, while you're at it, I would suggest breaking this extension up into two extensions, one per protocol adoption:

```swift
extension PreplannedMapModel: Equatable {
    nonisolated static func == (lhs: PreplannedMapModel, rhs: PreplannedMapModel) -> Bool {
        return lhs === rhs
    }
}

extension PreplannedMapModel: Hashable {
    nonisolated func hash(into hasher: inout Hasher) {
        hasher.combine(ObjectIdentifier(self))
    }
}
```

There's also the issue that neither of those conformances are abiding by their expected semantics (they're using identity, but identity is not the same as equality), but that's really outside the scope of this work.